guerrerocampos/pokedeck_capstone_app
# added some tests

Added Validations for Password and added more validations for username.

Steps to Test:
   Set Up:
       •Clone the Repo
       •$ git fetch --all
       •$ git checkout adding-some-tests
       •$ git pull origin adding-some-tests
       •$ rails s
       •on browser go to: localhost:3000
       • go to create account
       •try to create an account without a capital letter (should receive an error)
       •try to create an account without a special character (should receive an error)
       •try to create an account without a number (should receive an error)
       •try to create an account without a lowercase letter (should receive an error)
       •try to create an account with a 3 letter username and one with a 30 letter username (should receive an error)
       •try to create an account without all requirements met (should work)
